Add date when question was asked

NeoLou opened the pull request:
added dateAsked and lastUpdated to faq model

lukewarmtemp reviewed (approved):
LGTM!

Cali0707 reviewed (changes requested):
@NeoLou please read here: https://mongoosejs.com/docs/timestamps.html

This is an easier way to implement this functionality because it will allow us to not need to remember to update the lastUpdated field

NeoLou commented:
I changed the model to include timestamps to know the created date and updated date according to this documentation: https://mongoosejs.com/docs/timestamps.html

Cali0707 commented on server/src/models/FaqModel.js:
```diff
@@ -25,6 +25,7 @@ const FaqSchema = new Schema(
     },
   },
   { strict: true },
+  { timestamps: true },
```
```suggestion
  { strict: true, timestamps: true },
```
For the options for a schema, they need to all be in the same object
